multidim-interop/js/v0.41/test/ping.spec.ts

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable no-console */
+/* eslint-env mocha */
+
+import { } from 'aegir/chai'
+// import { createClient } from 'redis'
+
+import { createLibp2p, Libp2pOptions } from 'libp2p'
+// import { tcp } from '@libp2p/tcp'
+import { webSockets } from '@libp2p/websockets'
+import { noise } from '@chainsafe/libp2p-noise'
+import { mplex } from '@libp2p/mplex'
+import { yamux } from '@chainsafe/libp2p-yamux'
+import { multiaddr } from '@multiformats/multiaddr'
+
+describe('ping test', () => {
+  it('should ping', async () => {
+    let nodeFetchImport = "node-fetch"
+    let fetch = (typeof window === "undefined") ?
```

Comment:
I believe stuff like https://www.npmjs.com/package/cross-fetch can handle this for you

Reply:
Node 18 has fetch available as a global, no need to add extra deps if you're running on that.